src/main/webapp/components/directives/slider/slider.js

```diff
@@ -0,0 +1,92 @@
+(function () {
+  'use strict';
+
+  var module = angular.module('ispn.directives.slider', []);
+
+  module.directive('slider', function () {
+      return {
+        restrict: 'E',
+        scope: {
+          min: '=',
+          max: '=',
+          unit: '=',
+          step: '@',
+          value: '=',
+          from: '=',
+          to: '='
+        },
+        templateUrl: 'components/template/slider/slider.html',
+        link: function (scope, element, attrs) {
+          var fromParsed = null, toParsed = null;
+          var sliderElement = $(element).find('#slider');
+          var sliderEvent = 'slide';
+          var twoValueSlider = ispn.util.isNotNullOrUndefined(scope.from) && ispn.util.isNotNullOrUndefined(scope.to);
+          var oneValueSlider = ispn.util.isNotNullOrUndefined(scope.value);
+          if (twoValueSlider) {
+            sliderElement.noUiSlider({
+              start: [
+                  scope.from || scope.min,
+                  scope.to || scope.max
+              ],
+              step: parseFloat(scope.step || 1),
+              connect: true,
+              range: {
+                min: [parseFloat(scope.min)],
+                max: [parseFloat(scope.max)]
+              }
+            });
+
+            //event callback when user touches slider
+            sliderElement.on(sliderEvent, function () {
+              fromParsed = parseFloat(sliderElement.val()[0]);
+              toParsed = parseFloat(sliderElement.val()[1]);
+              return scope.$apply(function () {
+                scope.from = fromParsed;
+                scope.to = toParsed;
+              });
+            });
+
+            //watch model for changes
+            scope.$watch('from', function (newVal, oldVal) {
+              if (newVal !== fromParsed) {
```

Comment:
Here I would rather use scope.from instead of fromParsed -> it seems more obvious what is happenning.